main.js

```diff
@@ -194,18 +203,81 @@ function createWindow () {
   });
 }
 
+function createTrayIcon() {
+  logger.info('Creating tray icon');
+
+  // TODO: better to not use a hardcoded icon file
+  tray = new Tray(path.join(__dirname, '/images/icon_48.png'));
+
+  trayContextMenu = Menu.buildFromTemplate([
+      {id: 'show', label: locale.messages.show.message, visible: !mainWindow.isVisible(), click: () => {showWindow();}},
+      {id: 'hide', label: locale.messages.hide.message, visible: mainWindow.isVisible(), click: () => {hideWindow();}},
+      {id: 'quit', label: locale.messages.quit.message, click: () => {app.quit();}}
+  ]);
+
+  // NOTE: a function equivalent to the following will become part of the
```

Comment:
Maybe open a "remind this" Issue here to remove it when electron time comes?

Reply:
Yes, an issue to track this, linked in the code, would be great. That issue can link to the places we can go to see if it's available yet.